pyconcz/templates/pages/ostrava.html

```diff
@@ -127,98 +200,176 @@ <h2>Travelling around Ostrava</h2>
+                        <dd>
+                            The easiest way! Inside the tram, bus or trolleybus, just tap your card on a yellow machine and wait for the green light. Your ticket is purchased. Anytime you do this, the yellow machines are tracking your card and will use the cheapest price possible for your ticket.
+                        </dd>
+                        <dt>Cash</dt>
+                        <dd>
+                            Buy tickets for 30 min, 24 hours or 3 days in yellow ticket machines located at some tram stops or in news stands.
+                            <br>
+                            Don’t forget to punch (stamp) the ticket right after you enter the vehicle.
+                            Once stamped, the ticket is valid for all public transport in Ostrava within the ticket’s time period.
+                        </dd>
+                        <dt>Text message (SMS)</dt>
+                        <dd>
+                            If you have a Czech phone number, send the text “DPO70” to <a href="tel:90230">90230</a> to buy a 70-minute ticket for 30 CZK.
+                            <i>(This is a special service by Czech operators, so unfortunately it does not work from international numbers.)</i>
+                        </dd>
+                    </dl>
                 </div>
                 <div class="col">
                     <h3>Bicycle rental</h3>
-                    <p>You can try shared bikes by <a href="https://nextbike.pl/en/">Nextbike</a> available right on streets! Download “Nextbike” <a href="https://play.google.com/store/apps/details?id=de.nextbike&hl=en">Android</a> or <a href="https://itunes.apple.com/us/app/nextbike/id504288371?mt=8">iOS application</a>.</p>
-                    <p>Use a <a href="https://ostrava.dopracenakole.net/#hledani@trasa=64325101@plan=balanced">specialized map</a> to plan your bike trips because Google Maps don’t offer cycling navigation for Czech Republic.</p>
-
+                    <p>
+                        Pink bikes from <a href="https://www.rekola.cz/en/">Rekola Bikesharing</a> are available right on streets across the city.
+                        Download the app for <a href="https://play.google.com/store/apps/details?id=cz.rekola.app&amp;hl=cs">Android</a> or <a href="https://itunes.apple.com/cz/app/rekola/id888759232?mt=8">iOS</a>.
+                    </p>
+                    <p>
+                        Another option is <a href="https://nextbike.pl/en/">Nextbike</a> (<a href="https://play.google.com/store/apps/details?id=de.nextbike&amp;hl=en">Android</a>, <a href="https://itunes.apple.com/us/app/nextbike/id504288371?mt=8">iOS</a>).
+                    </p>
+                    <p>
+                        Use a <a href="https://ostrava.dopracenakole.net/#hledani@trasa=64325101@plan=balanced">specialized map</a> to plan your bike trips because Google Maps don’t offer cycling navigation for Czech Republic.
```

Comment:
```suggestion
                        Use a <a href="https://ostrava.dopracenakole.net/#hledani@trasa=64325101@plan=balanced">specialized map</a> to plan your bike trips because Google Maps don’t offer cycling navigation for the Czech Republic.
```